### PR TITLE
Support OTA Updates

### LINKED ISSUE #38: Remove Tor
Now that Tor is dockerized, we can remove it from the OS after https://github.com/getumbrel/umbrel-compose/pull/28 is merged

### LINKED ISSUE #40: OTA Updates
Update Umbrel OS Over-The-Air 🚀

### PR DESCRIPTION
**Pull request by mayankchhabra**
Closes #40 
Also resolves #38

### CODE REVIEW

File: stage2/01-sys-tweaks/files/rc.local [hunk 1]
```diff
@@ -39,22 +39,10 @@ if [ ! -f /home/umbrel/statuses/disk-partitioned ]; then
...
-        RPCPASS=`cat /home/umbrel/secrets/rpcpass.txt`
-        sed -i "s/RPCPASS/${RPCPASS}/g;" /home/umbrel/docker-compose.yml
-        sed -i "s/RPCPASS/${RPCPASS}/g;" /home/umbrel/lnd/lnd.conf
 
-        # Making this as done so we don't go through updates again
-        touch /home/umbrel/statuses/service-configured
-        chown -R umbrel.umbrel /home/umbrel/statuses/service-configured
-    fi
+    cd /home/umbrel
+    export REGTEST=1
```
Comment (mayankchhabra) [inline stage2/01-sys-tweaks/files/rc.local:44]: Remove REGTEST before merge

File: stage2/05-docker-compose/01-run.sh [hunk 2]
```diff
@@ -6,7 +6,7 @@ echo "Installing docker-compose from pip3, and also setting up the box folder st
 on_chroot << EOF
 pip3 install docker-compose
 cd /home/${FIRST_USER_NAME}
-wget -qO- "https://raw.githubusercontent.com/getumbrel/umbrel-compose/master/install-box.sh" | sh
+wget -qO- "https://raw.githubusercontent.com/mayankchhabra/umbrel-compose/feature-ota/install-box.sh" | sh
```
Comment (mayankchhabra) [inline stage2/05-docker-compose/01-run.sh:9]: Change to `getumbrel/umbrel-compose` repo once `feature-ota` branch of mayankchhabra/umbrel-compose is merged

**Comment by mayankchhabra** [comment]
No additional configuration is required to support OTA updates now that we've got `karen`! (#47)

**Comment by nolim1t** [comment]
would-like-to-speak-with-umbrel-manager

Would like to speak with ```umbrel-manager``` !

**Comment by mayankchhabra** [comment]
lool, the pieces are automatically falling in place

**Pull request closed without merging**